cmd/storj-sim/network.go

```diff
@@ -264,6 +264,11 @@ func newNetwork(flags *Flags) (*Processes, error) {
 			)
 		}
 
+		if flags.Redis != "" {
```

Comment:
I'm not sure if this is correct

Reply:
storj-sim network --redis="redis://127.0.0.1:6379?db=0" setup fails to create a config file